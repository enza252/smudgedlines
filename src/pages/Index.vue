<template>
  <v-container fluid class="d-flex flex-column flex-grow-1">
    <v-row class="d-flex flex-grow-1" justify="center" align="center">
      <Header/>
    </v-row>
    <v-row class="d-flex flex-column flex-grow-0">
      <Splash/>
    </v-row>
  </v-container>
</template>

<script>
import Header from '@/views/Header/Header'
import Splash from '@/views/Splash/Splash'

export default {
  name: 'Index',
  components: {
    Header,
    Splash
  }
}
</script>

<style>
</style>
