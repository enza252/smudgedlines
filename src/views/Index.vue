<template>
  <div class="wrapper">
    <parallax
      class="page-header header-filter"
      :style="headerStyle"
    >
      <div class="md-layout">
        <div class="md-layout-item">
          <div class="brand">
            <h1>Smudged Lines</h1>
            <h3>Handcrafted designs using locally sourced & produced fabric.</h3>
          </div>
        </div>
      </div>
    </parallax>
    <div class="main main-raised">
      <div class="section section-basic">
        <div class="container">
          <div class="title">
            <h2 id="designs">
              Designs
            </h2>
          </div>
          <designs-section />
        </div>
      </div>
    </div>
  </div>
</template>

<script>
import DesignsSection from '@/views/components/DesignsSection'

export default {
  name: 'Index',
  components: {
    DesignsSection
  },
  bodyClass: 'index-page',
  props: {
    image: {
      type: String,
      default: require('@/assets/img/bg-acid-synth.png')
    }
  },
  data () {
    return {
      firstname: null,
      email: null,
      password: null,
      leafShow: false
    }
  },
  computed: {
    headerStyle () {
      return {
        backgroundImage: `url(${this.image})`
      }
    },
    signupImage () {
      return {
        backgroundImage: `url(${this.signup})`
      }
    }
  },
  mounted () {
    this.leafActive()
    window.addEventListener('resize', this.leafActive)
  },
  beforeDestroy () {
    window.removeEventListener('resize', this.leafActive)
  },
  methods: {
    leafActive () {
      if (window.innerWidth < 768) {
        this.leafShow = false
      } else {
        this.leafShow = true
      }
    }
  }
}
</script>
<style lang="scss">
.section-download {
  .md-button + .md-button {
    margin-left: 5px;
  }
}

@media all and (min-width: 991px) {
  .btn-container {
    display: flex;
  }
}
</style>
