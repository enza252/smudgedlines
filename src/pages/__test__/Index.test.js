import Index from '@/pages'
import { render } from '@testing-library/vue'
import Vuetify from 'vuetify'

const options = {
  routes: [],
  vuetify: new Vuetify()
}

let fragment

beforeAll(async () => {
  const { asFragment } = await render(Index, options)
  fragment = asFragment
})

describe('tests the index page renders', () => {
  it('performs a snapshot test', () => {
    expect(fragment).toMatchSnapshot()
  })
})
