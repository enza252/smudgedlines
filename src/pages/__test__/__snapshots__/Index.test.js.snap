// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests the index page renders performs a snapshot test 1`] = `undefined`;
